awx_collection/tests/integration/targets/tower_inventory_source_update/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: Generate a test ID
+  set_fact:
+    test_id: "{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+
+- name: Generate names
+  set_fact:
+    project_name: "AWX-Collection-tests-tower_inventory_source_update-project-{{ test_id }}"
+    inv_name: "AWX-Collection-tests-tower_inventory_source_update-inv-{{ test_id }}"
+    inv_source1: "AWX-Collection-tests-tower_inventory_source_update-source1-{{ test_id }}"
+    inv_source2: "AWX-Collection-tests-tower_inventory_source_update-source2-{{ test_id }}"
+    inv_source3: "AWX-Collection-tests-tower_inventory_source_update-source3-{{ test_id }}"
+    org_name: "AWX-Collection-tests-tower_inventory_source_update-org-{{ test_id }}"
+
+
+- block:
+
+    - name: "Create a new organization"
+      tower_organization:
+        name: "{{ org_name }}"
+      register: created_org
+
+    - name: Create a git project without credentials
+      tower_project:
+        name: "{{ project_name }}"
+        organization: "{{ org_name }}"
+        scm_type: git
+        scm_url: https://github.com/ansible/test-playbooks
+        wait: true
+
+    - name: Create an Inventory
+      tower_inventory:
+        name: "{{ inv_name }}"
+        organization: "{{ org_name }}"
+        state: present
+
+    - name: Create another inventory w/ same name
+      tower_inventory:
+        name: "{{ inv_name }}"
+        organization: Default
+        state: present
+      register: created_inventory
+
+    - name: Create an Inventory Source
+      tower_inventory_source:
+        name: "{{ inv_source1 }}"
+        source: scm
+        source_project: "{{ project_name }}"
+        source_path: inventories/inventory.ini
+        description: Source for Test inventory
+        organization: "{{ created_org.id }}"
```

Comment:
If the next task is for testing org-based lookup, I think it makes sense to not provide `organization:` here. The inventory name should be sufficiently unique.

Reply:
`organization` is now a required parameter for the inventory source module so I can't take this out.  I forgot to update the name for the next task in this playbook.

Reply:
I’ve updated the module so that the org parameter isn’t required anymore, but I _do_ have this integration test create two same-named inventories associated with two separate orgs.  Thus, if I don’t specify the organization here, too many results get returned from the API.